source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -182,6 +182,41 @@ def body_incoming_wrench(env: ManagerBasedEnv, asset_cfg: SceneEntityCfg) -> tor
     return link_incoming_forces.view(env.num_envs, -1)
 
 
+def grab_images(
+    env: ManagerBasedEnv,
+    sensor_cfg: SceneEntityCfg = SceneEntityCfg("tiled_camera"),
```

Comment:
NIT: why default to tiled camera?
```suggestion
    sensor_cfg: SceneEntityCfg = SceneEntityCfg("tiled_camera"),
```

Reply:
Default to tiled as it is the most performant camera that can handle multiple dynamic objects, seemed like the obvious choice at the time but down to remove the default arg